README.rst

```diff
@@ -133,50 +145,82 @@ environment variables:
      export ARBEITSZEIT_APP_SERVER_NAME=localhost:5000
      export ARBEITSZEITAPP_TEST_DB="postgresql://postgres@localhost:5432/<name of test database>"
 
-After configuring the database connection you need to run the database
+If you are using Nix instead of ``venv``, you can safely omit four of 
```

Comment:
I am not so sure about this statement. In my experience is necessary to either set the same environment variables as shown above when using nix or use a custom settings file á la `custom_settings.py`.

Reply:
Yes!  You are right!  Sorry --- I did not fully understand how the settings files worked.  Hopefully, my corrected version is reasonably accurate.  Please let me know -- and thank you!